lib/ansible_tower_client/base_models/project_update.rb

```diff
@@ -1,4 +1,7 @@
 module AnsibleTowerClient
   class ProjectUpdate < BaseModel
+    def result_stdout
+      api.get(related['stdout']+'?format=txt').body
```

Comment:
I would prefer we default the format, but allow the caller to provider an override if needed.  Would be consistent with what we do for the Job class: https://github.com/ansible/ansible_tower_client_ruby/blob/master/lib/ansible_tower_client/base_models/job.rb#L14